crux-bench/src/crux/bench/sorted_maps_microbench.clj

```diff
@@ -0,0 +1,28 @@
+(ns crux.bench.sorted-maps-microbench
```

Comment:
Might be worth adding a `-main` to this which selects a default node to run it on (So you can quickly run it using `lein run -m crux.bench.sorted_maps_microbench`)?

Reply:
I'm trying to get away from requiring `lein run` for these - the feedback loop time's much quicker from the REPL - `(run-sorted-map-microbench (user/crux-node))` gets you the results much faster

Reply:
Seems fair. On top of that, I would suggest adding a note about it (and the general method to run it) to the `crux-bench` README.